### ISSUE TITLE
A11y_Azure Data Studio Nov2023_Files_Add Database File_VoiceOver: Name is not defined descriptively for Path section "More Option" button.

### ISSUE THREAD

**Issue by Mana-swi** [microsoft/azuredatastudio#25157]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #WCAG4.1.2; #NameRoleValue; #FTP; #SH_Azure Data Studio_Nov2023_Mac_Dec2023; #SQL Azure Data Studio; #MAC; #VoiceOver;

### Environment Details:
Application Name: Azure Data Studio
Operating System: macOS Sonoma 14.2.1
Azure data studio 1.47.0
VoiceOver

### Repro Steps:

1. Install Azure data studio application and open it
2. Make sure to enable "Preview" feature in settings.
3. Turn on voiceover.
4. Tab till "Database" and press ENTER to expand the tree.
5. Tab till any database and pressF10 to open context menu.
6. Tab till "Properties Preview" and press ENTER.
7. Tab till "Files" tab and press ENTER.
8. Tab till "Database files" table and press ENTER to select the row.
9. Tab till "Add" button and press ENTER.
10. "Add Database File" dialog will appear.
11. Tab till " more option" button in Path Section.
12. You will observe that voice over is announcing as "..., button" for more option button.

### Actual Result:
Voice over is announcing as "..., button" for more option button present in "Path" section of "Add Database File" dialog.

Note: Similar issue is observed for Edit Database File ->Path section "More Option" button.

### Expected Result:
Voice over should announce as "Path More option button" for button present in "Path" section of "Add Database File" dialog. The name for "more option" button should be described clearly to understand the purpose of control. 

### User Impact:
Screen Reader users will face difficulties to understand the purpose of the button if name is not descriptive enough.

### Attachment:
![Name is not descripyive for More option button](https://github.com/microsoft/azuredatastudio/assets/155441466/0d00f153-5c89-4e34-9524-24e988c664d2)

**Comment by barbaravaldez** [comment]
Fixed by https://github.com/microsoft/azuredatastudio/pull/25233

**Comment by corivera** [comment]
@Mana-swi The issue has been fixed in the latest Mac Insiders build of ADS [here](https://azuredatastudio-update.azurewebsites.net/latest/darwin-universal/stable). Can you verify that the issue is resolved?

**Comment by Mana-swi** [comment]
Verified the issue in below environment
Version: 1.48.0-insider (Universal)
VS Code: 1.82.0
Electron: 25.9.8
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 23.2.0
MacOS: sonoma 14.2.1

The issue is fixed, hence closing the bug. Please find attachment for reference. Thanks!

![Browse file button](https://github.com/microsoft/azuredatastudio/assets/155441466/9149b27a-77d7-4458-92d3-6c2581f4d446)
